lib/Parser/rterrors.h

```diff
@@ -374,3 +374,4 @@ RT_ERROR_MSG(WASMERR_InvalidGlobalRef, 5673, "", "Global initialization does not
 RT_ERROR_MSG(WASMERR_NeedMemoryObject, 5674, "%s is not a WebAssembly.Memory", "WebAssembly.Memory object expected", kjstTypeError, 0)
 RT_ERROR_MSG(WASMERR_InvalidTypeConversion, 5675, "Invalid WebAssembly type conversion %s to %s", "Invalid WebAssembly type conversion", kjstTypeError, 0)
 RT_ERROR_MSG(WASMERR_DivideByZero, 5676,    "",  "Division by zero", kjstError, 0)
+RT_ERROR_MSG(JSERR_CantRedefineProp, 5677, "Cannot redefine property '%s'", "Cannot redefine property", kjstTypeError, 0)
```

Comment:
>JSERR_CantRedefineProp [](start = 13, length = 22)

How is this different from error message JSERR_DefineProperty_NotConfigurable (Cannot redefine non-configurable property). Are there cases when we hit this error and the property is configurable?